.github/workflows/ci-native.yml

```diff
@@ -13,12 +13,16 @@ on:
 
 jobs:
   build:
+    # TODO: It would be nice to move away from Ubuntu 22.04, but that would
```

Comment:
The webkit-container-sdk is a 24.04 image with the dependencies :)

Can make an image based on it with webkit built.

Reply:
Yes, we could but GitHub self-hosted runners won't take an arbitrary image. That would mean switching over to our own runner, so I would rather do it in separately.